youtube/templates/watch.html

```diff
@@ -559,6 +568,17 @@ <h2 class="title">{{ title }}</h2>
             </ul>
         </details>
 
+        {% if endcard %}
+        <details class="endcard-dropdown">
+            <summary>Endcard</summary>
+            <div class="dropdown-content">
+                {% for e in endcard %}
+                {{ common_elements.item(e) }}
+                <!-- {{ e }} -->
+                {% endfor %}
+            </div>
+        </details>
+        {% endif %}
```

Comment:
Why not add the video at the top of the related videos similar to the layout in YouTube's "Watch Next" feature instead of adding another button to the interface?